src/helpers/formatters.ts

```diff
@@ -0,0 +1,16 @@
+import {Balance} from '@app/types';
+import {WEI} from '@app/variables/common';
+
+export const formatBalanceToNumber = (balance: Balance): number => {
+  return Number(balance.toString());
+};
+
+export const formatBalanceWithWEI = (
+  balance: Balance | number | string,
+): number => {
+  if (!balance) {
+    return 0;
+  }
+
+  return Number(balance.toString()) / WEI;
```

Comment:
` new Decimal(balance.toString()).div(WEI)` imho these solution will be better

Reply:
Fixed,  but it would be nice if you revealed the details why